ui/apps/platform/src/Containers/Collections/CollectionFormModal.tsx

```diff
@@ -90,35 +103,32 @@ function CollectionsFormModal({
     } else if (data) {
         content = (
             <CollectionFormDrawer
-                // We do not want to present the user with options to change the collection when in this modal
-                hasWriteAccessForCollections={false}
-                action={{
-                    type: 'view',
-                    collectionId,
-                }}
+                hasWriteAccessForCollections={hasWriteAccessForCollections}
```

Comment:
Questions for my understanding. If role has **write** access:
1. For view action, you can click **Edit** button to open collection in a new page.
    * Is this also possible in reporting?
2. For create action, it is implied.
    * That is, create and read-only access is a contradiction?
    * You can create **only** in reporting, **not** in collections?

Reply:
1. This is not possible in reporting. The only path to the collection form from the reporting page place the form in "create" mode where the edit/clone/delete menu buttons are not applicable. Other collections that are linked below in the form are marked to open a new tab on the individual collection's page:
![image](https://user-images.githubusercontent.com/1292638/210407766-0459606e-9812-4e45-9b69-67ce02787d42.png)
2. 
- An action of "create" when the user only has read access is a contradiction that is currently only enforced in FE logic. This is unfortunately currently not enforced in the type system. 
- Correct, the collection modal is not used for creation via the main collection pages, only in VM reporting.